packages/orchestrator-ui-components/src/components/WFOStartPage/WFOListItemStartPage.tsx

```diff
@@ -3,7 +3,7 @@ import { EuiFlexGroup, EuiFlexItem, EuiIcon, EuiTextColor } from '@elastic/eui';
 import moment from 'moment';
 import { Process, ProductBase } from '../../types';
 
-interface Subscription {
+interface WFOSubscription {
```

Comment:
This might be an accidental rename, it should still be Subscription.

Reply:
Ok. Fixed